rxkeystore/src/main/java/com/nexenio/rxkeystore/provider/cipher/asymmetric/ec/EcCipherProvider.java

```diff
@@ -38,6 +38,14 @@ public EcCipherProvider(RxKeyStore rxKeyStore) {
         return Single.fromCallable(() -> new ECGenParameterSpec(CURVE_NAME));
     }
 
+    /**
+     * Note: keyPurposes are not being used here.
```

Comment:
Why not?

Reply:
It is not implemented in the key generators used by us or default.